.gitlab-ci.yml

```diff
@@ -1185,6 +1187,24 @@ deploy_puppy:
     - export PACKAGE_VERSION=$(inv agent.version --url-safe)
     - aws s3 cp --region us-east-1 ./agent $S3_DSD6_URI/puppy/agent-$PACKAGE_VERSION --grants read=uri=http://acs.amazonaws.com/groups/global/AllUsers full=id=3a6e02b08553fd157ae3fb918945dd1eaae5a1aa818940381ef07a430cf25732
 
+# deploy process agent and system-probe to staging bucket
+deploy_process_and_sysprobe:
+  stage: internal_deploy
+  when: manual
+  image: 486234852809.dkr.ecr.us-east-1.amazonaws.com/ci/datadog-agent-builders/deploy:latest
+  before_script:
+    - cd $OMNIBUS_PACKAGE_DIR
+    - ls
+  tags: [ "runner:main", "size:large" ]
+  script:
+    - dpkg -x datadog-agent_*_amd64.deb ./out
+    # Use tag or shortened branch with short commit hash to identify the binary
+    - export SHORT_REF=$(echo $CI_COMMIT_REF_NAME | cut -d'/' -f2- | cut -c -10 | sed -E 's/[^[:alnum:]]+/-/g')
+    - export NAME="${CI_COMMIT_TAG:-$SHORT_REF}-${CI_COMMIT_SHA:0:7}"
```

Comment:
Do you have any examples of what these names will look like?

Reply:
```process-agent-amd64-process-ag-5c6a4e2```

Reply:
This is a messy name. Any reason we can't just use `process-agent-amd64-5c6a4e2`

Reply:
Looks like @leeavital reccommended adding the ref. What use is it?